README.md

```diff
@@ -425,6 +429,36 @@ def xor(*args: typing.Union[bytes, typing.Iterable[bytes]]) -> bytes:
     )
 ```
 
+## Basic Math
+
+### Modular Arithmetic
```

Comment:
lets add link to official operator 

https://docs.python.org/3.10/reference/datamodel.html#emulating-numeric-types
https://docs.python.org/3.10/library/operator.html#operator.mod

Reply:
good call, will do.